db/migrations/20240529151056_Page.sql

```diff
@@ -0,0 +1,9 @@
+-- Ensure the UUID extension is enabled
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE Page (
+    page_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+    user_id Uuid UNIQUE,
```

Comment:
I think we probably do NOT want to make this unique, or else I believe a User will only be able to add a single Page to the DB!

And I also think we want to add a Foreign Key Contraint here! I believe you can do that by adding the FK constraint with a `REFERENCES` after this column!

```
user_id UUID REFERENCES Users (user_id)
```

What that does is have Postgres make sure we get the `uuids` right! So if we tried to create a Page and gave it a `user_id` that doesn't exist in `Users#user_id` it will error out for us!

Reply:
And we probably want to add `NOT NULL` here too, to make sure that every page is owned by a User